### ISSUE TITLE
Add status to jobs

### ISSUE THREAD

**Issue by yannickperrenet** [orchest/orchest#51]
Use 
```
{
  ...
  "total_number_of_pipeline_runs": int,
  "completed_pipeline_runs": int,
  ...
}
```
that get returned by GET `/experiments/<experiment_uuid>` to set the status of an experiment in the front-end (inside the table you see after clicking "Experiments" in the left-pane menu). It should be something like "5/9" meaning 5 out of 9 pipeline runs of the experiment have completed.

**Comment by TAM360** [comment]
I can work on this.

**Comment by yannickperrenet** [comment]
We have now renamed _Experiments_ to _Jobs_ and are about to introduce scheduled (cron-like) jobs, where the status as described here should probably be moved (if at all) to the view of a job.

So when opening a job from the jobs list we can show the number of pipeline runs of that particular job that have completed. This can be helpful for jobs that consist of many pipeline runs. For scheduled runs this probably makes less sense though.

@fruttasecca Could we add this in #115?

**Comment by fruttasecca** [comment]
>So when opening a job from the jobs list we can show the number of pipeline runs of that particular job that have completed. This can be helpful for jobs that consist of many pipeline runs. For scheduled runs this probably makes less sense though.

I think this might already be in there given the recent changes, @ricklamers ?

**Comment by yannickperrenet** [comment]
> I don't think we need to add another element in the GUI to specify the total number of runs.

I agree on this point, but shouldn't it show e.g. 6/10 to indicate that 6 out of the total 10 pipeline runs have finished executing? Think of it as a progress indication, like a progress bar.

**Comment by fruttasecca** [comment]
Implemented in d705323e0eec319749d4dabe3fa73178e72c9faa.
